src/Components/News/NewsList.js

```diff
@@ -1,32 +1,56 @@
-import React from 'react';
-import '../CardListStyles.css';
-
-const NewsList = () => {
-    const newsMock = [
-        {id: 2, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 1, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 3, name: 'Titulo de prueba', description: 'Descripcion de prueba'}
-    ];
-
-    return (
-        <div>
-            <h1>Listado de Novedades</h1>
-            <ul className="list-container">
-                {newsMock.length > 0 ?
-                    newsMock.map((element) => {
-                        return(
-                            <li className="card-info" key={element.id}>
-                                <h3>{element.name}</h3>
-                                <p>{element.description}</p>
-                            </li>
-                        )
-                    })
-                :
-                    <p>No hay novedades</p>
-                }
-            </ul>
-        </div>
-    );
+import React from 'react'
+import '../CardListStyles.css'
+import { listHasValues } from '../../Utils/validation'
+import Title from '../Title/Title'
+import CustomCard from '../Card/CustomCard'
+import { Container } from '@material-ui/core'
+const lastNewsMock = [
+  {
+    id: 626,
+    name: 'VI Jornada recreativa',
+    content:
+      'Se realizó la VI jornada recreativa de invierno, jovenes de todas las edades participarion en juegos y los más chicos presentaron una coreografía espectacular!',
+    image: 'https://ukrainer.net/wp-content/uploads/2019/11/11-99.jpg',
+  },
+  {
+    id: 629,
+    name: 'Jornada domigo juguetes 12',
+    content:
+      'VI jornada recreativa de invierno, jovenes de todas las edades participarion en juegos y los m&aacute;s chicos presentaron una core',
+    image: 'https://ukrainer.net/wp-content/uploads/2019/11/0-113.jpg',
+    user_id: null,
+  },
+  {
+    id: 637,
+    name: 'Actividades anuales',
+    content:
+      'Jornada recreativa de invierno, jovenes de todas las edades participarion en juegos y los m&aacute;s chicos presentaron una core',
+    image:
+      'https://www.kleineherzen.or.at/wp-content/uploads/Kids_Mutter_Kind_haus2019.png',
+  },
+]
+const NewsList = ({ lastNews }) => {
+  const ListHasValues = listHasValues(lastNewsMock)
```

Comment:
the name of the const should be on lower case

Reply:
This had already changed, I don't know what happened. Sorry. I do it again.

Reply:
Done again